synapse/storage/schema/main/delta/59/999create_registration_tokens.sql

```diff
@@ -0,0 +1,23 @@
+/* Copyright 2021 Callum Brown
```

Comment:
I assume the `999` in the filename is just a placeholder for now? :P

Reply:
yep :-)